featuretools/tests/primitive_tests/test_feature_serialization.py

```diff
@@ -38,14 +38,16 @@
 BUCKET_NAME = "test-bucket"
 WRITE_KEY_NAME = "test-key"
 TEST_S3_URL = "s3://{}/{}".format(BUCKET_NAME, WRITE_KEY_NAME)
-TEST_FILE = "test_feature_serialization_feature_schema_5.0.0_entityset_schema_4.0.0_version_2.json"
+TEST_FILE = "test_feature_serialization_feature_schema_6.0.0_entityset_schema_5.0.0.json"
 S3_URL = "s3://featuretools-static/" + TEST_FILE
 URL = "https://featuretools-static.s3.amazonaws.com/" + TEST_FILE
 TEST_CONFIG = "CheckConfigPassesOn"
 TEST_KEY = "test_access_key_features"
 
 
 def assert_features(original, deserialized):
+    original = sorted(original, key=lambda x: x.unique_name())
```

Comment:
Had to add this to get `test_deserialize_features_s3` (primitive_tests/test_feature_serialization.py) to pass. The features deserialized from S3 were not in the same order as the features generated from DFS.

Reply:
the upload documentation has a `sorted` step that may be causing the re-order.  try reuploading without that sorting?

Reply:
Gotcha. That worked